### PR TITLE
javadoc

### LINKED ISSUE #73: Add javadoc
AimWithLimelight, DriveAutoFieldOriented, and DriveAutoRobotOriented, don't have javadoc for the class.

### PR DESCRIPTION
**Pull request by Jack-Haefele**
closes #73

### CODE REVIEW

File: src/main/java/frc/robot/commands/AimWithLimelight.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import frc.robot.subsystems.Drive;
 import frc.robot.subsystems.Limelight;
 
+/** Aims with the limelight. */
```
Comment (Berdenson) [inline src/main/java/frc/robot/commands/AimWithLimelight.java:20]: add that the command aims towards some object.

**Review by Berdenson — CHANGES_REQUESTED** [review]
Look good enough.
Add this: https://github.com/frc937/robot2024/issues/73#issuecomment-1955220577

File: src/main/java/frc/robot/commands/drive/DriveFieldOrientedHeadingSnapping.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public class DriveFieldOrientedHeadingSnapping extends Command {
    * Drives the robot field oriented with heading snapping. All values are suppliers to make the
    * command more versatile.
    *
-   * @param xSupplier The joystick value for the Y axis. [-1, 1] left positive.
+   * @param xSupplier The joystick value for the x axis. [-1, 1] left positive.
```
Comment (Berdenson) [inline src/main/java/frc/robot/commands/drive/DriveFieldOrientedHeadingSnapping.java:38]: Jack it needs to be an uppercase X

**Review by Berdenson — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/drive/DriveFieldOrientedHeadingSnapping.java (hunk 2 above)
Comment (Jack-Haefele) [reply src/main/java/frc/robot/commands/drive/DriveFieldOrientedHeadingSnapping.java]: explode

**Review by Berdenson — APPROVED** [review]

**Pull request merged**